Bio/AlignIO/StockholmIO.py

```diff
@@ -389,7 +407,13 @@ def __next__(self):
                 elif line[:5] == '#=GC ':
                     # Generic per-Column annotation, exactly 1 char per column
                     # Format: "#=GC <feature> <exactly 1 char per column>"
-                    pass
+                    feature, text = line[5:].strip().split(None, 1)
+                    # Each feature key could be used more than once,
+                    # so store the entries as a list of strings.
```

Comment:
Is this correct? Wikipedia says "Do not use multiple lines with the same #=GC label."
So isn't it something like
```python
if feature in gc:
    raise ValueError("Duplicate feature for #=GC label:\n" + line)
else:
    gc[feature] = text
```

Reply:
Yes, that change should be implemented. I think my problem was with some files that didn't follow the specs.